src/Sylius/Behat/Context/Ui/Shop/LoginContext.php

```diff
@@ -285,6 +285,15 @@ public function iShouldNotBeAbleToChangeMyPasswordAgainWithTheSameToken(): void
     {
         $this->resetPasswordPage->tryToOpen(['token' => 'itotallyforgotmypassword']);
 
+        $this->iShouldNotBeAbleToChangeMyPassword();
```

Comment:
```suggestion
        $this->iShouldNotBeAbleToChangeMyPasswordWithThisToken();
```
Should fix the build